.buildkite/pipeline-resource-definitions/security-solution-quality-gate/kibana-serverless-security-solution-quality-gate-detection-engine.yml

```diff
@@ -0,0 +1,33 @@
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+apiVersion: backstage.io/v1alpha1
+kind: Resource
+metadata:
+  name: buildkite-pipeline-kibana-serverless-security-solution-quality-gate-detection-engine
+  description: "[MKI] Executes Cypress tests for the Detection Engine team"
+spec:
+  type: buildkite-pipeline
+  owner: 'group:security-engineering-productivity'
+  system: buildkite
+  implementation:
+    apiVersion: buildkite.elastic.dev/v1
+    kind: Pipeline
+    metadata:
+      name: kibana-serverless-security-solution-quality-gate-detection-engine
```

Comment:
I think these names, in the pipelines, should be replaced with something less programmer-like because this will be the main displayed name. If we follow previous practise, it should be something like "Kibana / Servereless / Security Solution Quality Gate / Detection engine" - > this will be slugified to `https://buildkite.com/elastic/kibana-serverless-security-solution-quality-gate-detection-engine`

Reply:
Done